src/test/java/com/smartbear/postman/script/PostmanScriptParserTest.java

```diff
@@ -38,27 +40,61 @@
     private WsdlProject project;
     private Assertable assertable;
 
+    List<TestAssertion> assertions;
+
+    GroovyScriptAssertion groovyAssertion;
+    ValidHttpStatusCodesAssertion validHttpStatusCodesAssertion;
+    InvalidHttpStatusCodesAssertion invalidHttpStatusCodesAssertion;
+    ResponseSLAAssertion responseSLAAssertion;
+    EqualsAssertion equalsAssertion;
+    SimpleContainsAssertion simpleContainsAssertion;
+
     @Before
     public void prepare() {
         project = new WsdlProject();
         assertable = mock(Assertable.class);
+
+        groovyAssertion = mock(GroovyScriptAssertion.class);
+        validHttpStatusCodesAssertion = mock(ValidHttpStatusCodesAssertion.class);
+        invalidHttpStatusCodesAssertion = mock(InvalidHttpStatusCodesAssertion.class);
+        responseSLAAssertion = mock(ResponseSLAAssertion.class);
+        equalsAssertion = mock(EqualsAssertion.class);
+        simpleContainsAssertion = mock(SimpleContainsAssertion.class);
+
+        when(assertable.addAssertion(GroovyScriptAssertion.LABEL)).thenReturn(groovyAssertion);
+        when(assertable.addAssertion(ValidHttpStatusCodesAssertion.LABEL)).thenReturn(validHttpStatusCodesAssertion);
+        when(assertable.addAssertion(InvalidHttpStatusCodesAssertion.LABEL)).thenReturn(invalidHttpStatusCodesAssertion);
+        when(assertable.addAssertion(ResponseSLAAssertion.LABEL)).thenReturn(responseSLAAssertion);
+        when(assertable.addAssertion(EqualsAssertion.LABEL)).thenReturn(equalsAssertion);
+        when(assertable.addAssertion(SimpleContainsAssertion.LABEL)).thenReturn(simpleContainsAssertion);
+
+        assertions = new ArrayList<>();
+        when(assertable.getAssertionList()).thenReturn(assertions);
+    }
+
+    @Test
+    public void newLineTokenIsFirst() {
+        //???
+        //Each token regular expression appended in the beginning with "\s*" pattern which includes new line pattern,
+        //so the NEW_LINE pattern must be the first in the pattern check list, so it must be the first in the enum.
+        assertTrue("The NEW_LINE pattern must be the first in the enum TokenType", TokenType.values()[0] == TokenType.NEW_LINE);
     }
 
     @Test
-     public void parsesSettingGlobalVariable() throws SoapUIException {
-        String script = "postman.setGlobalVariable(\"string1\", \"abc\");\\npostman.setGlobalVariable(\"string2\", \"def\"); ";
+    public void parsesSettingGlobalVariable() throws SoapUIException {
+        String script = "postman.setGlobalVariable(\"string1\", \"abc\");\npostman.setGlobalVariable(\"string2\", \"def\"); ";
```

Comment:
why you changed test pattern
it was 
**action1\\
action2**
Not it is 
**action1\n
action2**

Reply:
The new line symbol was wrong.

Reply:
Unfortunately it wasn't wrong.
It's correct symbol to move script on next line.
https://unix.stackexchange.com/questions/281309/shell-syntax-how-to-correctly-use-to-break-lines
https://askubuntu.com/questions/1199854/why-are-there-backslash-line-breaks-on-this-apt-install-command
https://stackoverflow.com/questions/3871332/how-to-tell-bash-that-the-line-continues-on-the-next-line
Please return it back or describe why you ignore this symbol.
It is clear that you don't understand how it worked.
Are these changes still correct?

Reply:
The Postman's JS code is not related to Bash and any other command line, so your example is not applicable here. Please, provide any examples of correct JS code where command separator is slash instead of colon - and then I agree with you. Now I'm sure that new pattern is correct.

Reply:
![image](https://user-images.githubusercontent.com/20772040/134132151-c6a88b87-82b9-476d-bbc7-86fba1d527bc.png)

Reply:
I guess it's a typo indeed. Were these tests failing before your changes?

Reply:
@antone-sb , no we don't have fails before and after.